### PR TITLE
Graphviz writer

### PR DESCRIPTION
**Pull request by samvasko**
outstanding
- [x] Actors
- [ ] Weird rendering of edges in JS version
- [x] ~Tags for parents~

### CODE REVIEW

**Comment by briggySmalls** [comment]
Can I assist you with this?

**Comment by briggySmalls** [comment]
As discussed, we can probably fix point 2: weird rendering of edges in JS version at a later date. This seems to be an issue with the JS compatibility rather than a bug in this implementation?

**Comment by samvasko** [comment]
Ok this should be all ready, the JS issue is still there. The actors are circles because there is no flexible way of making shapes in graphviz. 
iTerm2 DySHOu

**Comment by samvasko** [comment]
Pending #24 , there is no way of changing the backend without changing the code

**Comment by briggySmalls** [comment]
You should be good to go once you rebase over master.

In this MR you will need to update `archie/cli/archie/utils/utils.go` to add a case for a new writer name. I thought I'd go for all lowercase names, so in this case I'd say `graphviz`?

File: writers/graphviz.go [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+	scribe.WriteLine("node [shape=plaintext; margin=0; fontname=Helvetica]")
+}
+
+// Footer writes a footer in mermaid syntax.
+func (p GraphvizStrategy) Footer(scribe Scribe) {
+	scribe.UpdateIndent(-1)
+	scribe.WriteLine("}")
+}
+
+// Element writes an element in mermaid syntax.
```
[the remark this answers is not in the document]
Comment (samvasko) [reply writers/graphviz.go]: Ups

**Comment by samvasko** [comment]
Oh I got it we can use emoji for actors!

**Comment by samvasko** [comment]
Ok emoji for actors is a terrible idea

**Comment by samvasko** [comment]
Should be ready now!

**Comment by briggySmalls** [comment]
vibes

**Pull request merged**